src/simdprune_tables.h

```diff
@@ -1,5 +1,9 @@
 #ifndef SIMDJSON_SIMDPRUNE_TABLES_H
 #define SIMDJSON_SIMDPRUNE_TABLES_H
+
+
+#if SIMDJSON_IMPLEMENTATION_ARM64 || SIMDJSON_IMPLEMENTATION_HASWELL || SIMDJSON_IMPLEMENTATION_WESTMERE
```

Comment:
I don't particularly mind if we get rid of this when we're only doing fallback, but was there a compilation problem here?

Reply:
@jkeiser There was, indeed.

Reply:
It is the usual "unused xxx". It would not be a problem otherwise.

I don't think it is harmful. If you end up missing it, it won't take you very long to figure out how to change the code.